packages/chain-helpers/src/blockchain/Blockchain.ts

```diff
@@ -88,53 +88,45 @@ export function EXTRINSIC_FAILED(result: ISubmittableResult): boolean {
   return ErrorHandler.extrinsicFailed(result)
 }
 
-/**
- * Parses potentially incomplete or undefined options and returns complete [[Options]].
- *
- * @param opts Potentially undefined or partial [[Options]] .
- * @returns Complete [[Options]], with potentially defaulted values.
- */
-export function parseSubscriptionOptions(
-  opts?: Partial<SubscriptionPromise.Options>
-): SubscriptionPromise.Options {
-  const {
-    resolveOn = IS_FINALIZED,
-    rejectOn = (result: ISubmittableResult) =>
-      EXTRINSIC_FAILED(result) || IS_ERROR(result),
-    timeout,
-  } = { ...opts }
-
-  return {
-    resolveOn,
-    rejectOn,
-    timeout,
-  }
+function defaultResolveOn(): SubscriptionPromise.ResultEvaluator {
```

Comment:
Could this be a getter? So we could access it like `Blockchain.resolveOn`? Would it make sense? You could also take it with `const { resolveOn } = Blockchain`, if needed.